wallets/cosmostationextention.json

```diff
@@ -0,0 +1,71 @@
+{
+  "$schema": "../wallet.schema.json",
+  "wallet_name": "cosmostationextention",
+  "pretty_name": "cosmostation",
+  "website": "https://www.cosmostation.io/",
+  "git_repo": "https://github.com/cosmostation/cosmostation-chrome-extension",
```

Comment:
I get a 404. Meant to be: https://github.com/cosmostation/cosmostation-chrome-extension-client ?